docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/unified/index.md

```diff
@@ -0,0 +1,167 @@
+
+These are defined in the `selectors.yml` file ([source](https://github.com/snowplow/dbt-snowplow-web/blob/main/selectors.yml)) within the package, however in order to use these selections you will need to copy this file into your own dbt project directory. This is a top-level file and therefore should sit alongside your `dbt_project.yml` file. If you are using multiple packages in your project you will need to combine the contents of these into a single file.
+
+### 3. Check source data
+
+This package will by default assume your Snowplow events data is contained in the `atomic` schema of your [target.database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). In order to change this, please add the following to your `dbt_project.yml` file:
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__atomic_schema: schema_with_snowplow_events
+    snowplow__database: database_with_snowplow_events
+```
+:::info Databricks only
+
+Please note that your `target.database` is NULL if using Databricks. In Databricks, schemas and databases are used interchangeably and in the dbt implementation of Databricks therefore we always use the schema value, so adjust your `snowplow__atomic_schema` value if you need to.
+
+:::
+
+### 4. Enabled desired contexts
+
+The unified package has the option to join in data from the following Snowplow enrichments and out-of-the-box context entities:
+
+- [IAB enrichment](/docs/enriching-your-data/available-enrichments/iab-enrichment/index.md)
+- [UA Parser enrichment](/docs/enriching-your-data/available-enrichments/ua-parser-enrichment/index.md)
+- [YAUAA enrichment](/docs/enriching-your-data/available-enrichments/yauaa-enrichment/index.md)
+- Browser context
+- Mobile context
+- Geolocation context
+- App context
+- Screen context
+- Deep Link context
+- App Error context
+- Core Web Vitals
+- Consent (Preferences & cmp visible)
+
+By default these are **all disabled** in the unified package. Assuming you have the enrichments turned on in your Snowplow pipeline, to enable the contexts within the package please add the following to your `dbt_project.yml` file:
+
+```yml title="dbt_project.yml"
+vars:
+  snowplow_unified:
+    snowplow__enable_iab: true
+    snowplow__enable_ua: true
+    snowplow__enable_yauaa: true
+    snowplow__enable_browser_context: false
+    snowplow__enable_mobile_context: false
+    snowplow__enable_geolocation_context: false
+    snowplow__enable_application_context: false
+    snowplow__enable_screen_context: false
+    snowplow__enable_deep_link_context: false
+    snowplow__enable_consent: false
+    snowplow__enable_cwv: false
+    snowplow__enable_app_errors: false
+```
```

Comment:
Why do we only enable some?

Reply:
cause i prob realised half way there that i should not do this one by one then never finished